### ISSUE TITLE
YAML parsing fails with global installation

### ISSUE THREAD

**Issue by rliebz** [IBM/openapi-validator#405]
# Description

When referencing `@ibm-cloud/openapi-ruleset` in a `.spectral.yaml` file, parsing fails and the following message appears:

```
[Info] Problem reading Spectral config file, using default IBM Spectral ruleset. Cause for error:
ENOENT: no such file or directory, open '/Users/me/my-project@ibm-cloud/openapi-ruleset'
```

# Steps to Reproduce

## Local Machine

With the following `.spectral.yaml`:
```yaml
# Or `extends: '@ibm-cloud/openapi-ruleset'`
extends: [['@ibm-cloud/openapi-ruleset', 'all']]
rules:
  # The actual rules here don't affect anything
  string-boundary: false
```

Install any version of `ibm-openapi-validator` `>= 0.52.1`, and the ruleset globally:

```sh
npm -g install ibm-openapi-validator @ibm-cloud/openapi-ruleset
```

Run the command with the `--debug` flag on any file:

```sh
$ lint-openapi --debug --verbose -r .spectral.yaml openapi.yaml 

[Info] Problem reading Spectral config file, using default IBM Spectral ruleset. Cause for error:
ENOENT: no such file or directory, open '/Users/me/my-project/@ibm-cloud/openapi-ruleset'
```

## Docker

This is reproducible with the same `.spectral.yaml` as above and a one-liner running docker:

```
$ docker run --rm -tiv $PWD:/data:ro jamescooke/openapi-validator:0.58.1 --debug --verbose openapi.yaml  

[Info] Problem reading Spectral config file, using default IBM Spectral ruleset. Cause for error:
ENOENT: no such file or directory, open '/data/@ibm-cloud/openapi-ruleset'
```

## Additional information

This problem does not occur on `0.52.0`, but does occur on `0.52.1` and later.

It also does not occur if the same configuration is ported to Javascript:

```js
const ibmRuleset = require("@ibm-cloud/openapi-ruleset");

module.exports = {
  extends: [[ibmRuleset, 'all']],
  rules: {
    "string-boundary": false,
  },
};
```

**Comment by dpopp07** [comment]
The ruleset needs to be in the local filesystem for Spectral to find it, like any NPM dependency. If you haven't already, you need to install the ruleset package as a local dependency. Spectral will automatically find it in `node_modules/`.

We were a bit late getting documentation out on this but you can find more info in our [customization docs](https://github.com/IBM/openapi-validator/blob/main/docs/ibm-cloud-rules.md#customization). If you're still having trouble, let me know and we can debug

**Comment by rliebz** [comment]
Thanks for the quick response. I can replicate the behavior using the spectral CLI directly, so I'll follow up on their end. I appreciate your time here.
